scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -295,8 +295,8 @@ import org.apache.tools.ant.taskdefs.Java
 class ScalaTestAntTask extends Task {
   private var includes:  String = ""
   private var excludes:  String = ""
-  private var maxMemory: String = null
-  private var suffixes:  String = null
+  private var maxMemory: String | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.